src/codeu/chat/server/View.java

```diff
@@ -71,7 +70,17 @@ public View(Model model) {
   public Collection<Message> getMessages(Collection<Uuid> ids) {
     return intersect(model.messageById(), ids);
   }
-
+  
+  @Override
+  public ServerInfo getInfo(){
```

Comment:
You should delete this `getInfo` method since it conflicts with the `getInfo` method at the bottom of this class that was brought in when you merged master in.